### PR TITLE
Marcus/Replace all instances of person with task

### PR DESCRIPTION
**Pull request by MarcTzh**
Fix #99

### CODE REVIEW

File: docs/tutorials/AddRemark.md [hunk 1]
```diff
@@ -308,9 +308,9 @@ Just add [this one line of code!](https://github.com/se-edu/addressbook-level3/c
 **`PersonCard.java`:**
 
 ``` java
-public PersonCard(Person person, int displayedIndex) {
+public PersonCard(Person task, int displayedIndex) {
```
Comment (BobbyZhouZijian) [inline docs/tutorials/AddRemark.md]: This is still not changed

File: docs/tutorials/AddRemark.md [hunk 2]
```diff
@@ -340,23 +340,23 @@ save it with `Model#setPerson()`.
             throw new CommandException(Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
         }
 
-        Person personToEdit = lastShownList.get(index.getZeroBased());
-        Person editedPerson = new Person(personToEdit.getName(), personToEdit.getPhone(), personToEdit.getEmail(),
-                personToEdit.getAddress(), remark, personToEdit.getTags());
+        Person taskToEdit = lastShownList.get(index.getZeroBased());
```
Comment (BobbyZhouZijian) [inline docs/tutorials/AddRemark.md]: The class here is still not changed to Task

File: src/main/java/seedu/address/storage/JsonAdaptedTask.java [hunk 3]
```diff
@@ -31,12 +31,12 @@
...
     /**
-     * Constructs a {@code JsonAdaptedPerson} with the given person details.
+     * Constructs a {@code JsonAdaptedTask} with the given task details.
      */
     @JsonCreator
-    public JsonAdaptedPerson(@JsonProperty("name") String name, @JsonProperty("phone") String phone,
-            @JsonProperty("email") String email, @JsonProperty("address") String address,
-            @JsonProperty("tagged") List<JsonAdaptedTag> tagged) {
+    public JsonAdaptedTask(@JsonProperty("name") String name, @JsonProperty("phone") String phone,
+                           @JsonProperty("email") String email, @JsonProperty("address") String address,
```
Comment (BobbyZhouZijian) [inline src/main/java/seedu/address/storage/JsonAdaptedTask.java]: Why extra indent here?

**Review by BobbyZhouZijian — COMMENTED** [review]
Generally looks good to me. Just have a few comments below. 

Need to change some other things to tasks in the future as well.

File: docs/tutorials/AddRemark.md (hunk 1 above)
Comment (MarcTzh) [reply docs/tutorials/AddRemark.md]: These are in addremark.md not actual code, not sure if it needs to be changed

File: docs/tutorials/AddRemark.md (hunk 2 above)
Comment (MarcTzh) [reply docs/tutorials/AddRemark.md]: same reply as above

File: docs/tutorials/AddRemark.md (hunk 1 above)
Comment (MarcTzh) [reply docs/tutorials/AddRemark.md]: ok undid changes in these documents, please review again

File: docs/tutorials/AddRemark.md (hunk 2 above)
Comment (MarcTzh) [reply docs/tutorials/AddRemark.md]: ok undid changes in these documents, please review again

**Review by BobbyZhouZijian — APPROVED** [review]
LGTM now

**Pull request merged**